Google-Maps-iOS-Utils.podspec

```diff
@@ -21,7 +21,7 @@ Pod::Spec.new do |s|
   s.module_name = "GoogleMapsUtils"
   s.swift_version = '5.0'
 
-  s.dependency 'GoogleMaps', '~> 7.3'
+  s.dependency 'GoogleMaps', '~> 8.3'
```

Comment:
Since 8.3 does introduce some potentially breaking changes (new initializers, etc), would it make sense to define this as `~> 8.0` to allow clients a bit more flexibility?

Reply:
Correct; updated this!